app/queries/keywords_query.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class KeywordsQuery
+  attr_reader :keywords, :filter_params
+
+  def initialize(keywords, filter_params)
+    @keywords = keywords
+    @filter_params = filter_params
+  end
+
+  def call
+    return nil unless filter_params[:adwords_url_contains]
```

Comment:
We can simplify this.

```suggestion
    return unless filter_params[:adwords_url_contains]
```